extensions/wikia/Blogs/tests/BlogsHelperTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+class BlogsHelperTest extends WikiaBaseTest {
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../Blogs.php';
+		parent::setUp();
+	}
+
+	/**
+	 * SUS-1635: Regression test for BlogsHelper::AbortMove hook handler
+	 * Verify blog pages and comments can be renamed, pages can be moved into blog namespace and that pages can't be moved
+	 * out of blogs namespaces or into comment namespace
+	 *
+	 * @covers BlogsHelper::onAbortMove()
+	 * @dataProvider blogsMoveRestrictionsDataProvider
+	 * @param int $sourceNamespace
+	 * @param int $targetNamespace
+	 * @param bool $shouldAllow
+	 */
+	public function testBlogsMoveRestrictions( int $sourceNamespace, int $targetNamespace, bool $shouldAllow ) {
+		/** @var Title|PHPUnit_Framework_MockObject_MockObject $sourceTitleMock */
+		$sourceTitleMock = $this->getMock( Title::class, [ 'getNamespace' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getNamespace" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)